app/containers/Vaults/index.js

```diff
@@ -487,41 +487,39 @@ const VaultsWrapper = (props) => {
     if (vault.pureEthereum) {
       vaultKey = `${vault.address}-eth`;
     }
-    let banner = null;
-    let decimals = 18;
-    if (vault && vault.token && vault.token.decimals > 0) {
-      decimals = setDecimals(vault.token.decimals);
-    }
-
-    if (
-      vault &&
-      vault.balanceOf &&
-      vault.balanceOf[0].value >= 10 ** decimals &&
-      isMigrated &&
-      !hasBeenShown
-    ) {
-      banner = <MigrationBannerSvg vaults={migratedVaultsWithBalance} />;
-      hasBeenShown = true;
-    }
```

Comment:
Completely refactor banner logic.

Previous logic:
- Conditionally render banner instead every migration vault

New logic:
- Only attempt to render banner once, at the top of migration vaults